virttest/libvirt_xml/vm_xml.py

```diff
@@ -563,17 +563,18 @@ def del_controller(self, controller_type=None):
         for controller in del_controllers:
             self.xmltreefile.remove(controller)
 
-    def get_controllers(self, controller_type=None):
+    def get_controllers(self, controller_type=None, model=None):
         """
-        Get controllers according controller type
+        Get controllers according controller type and/or model type
 
         :param controller_type: type of controllers need to get
         :return: controller list
         """
         all_controllers = self.xmltreefile.findall("devices/controller")
         type_controllers = []
         for controller in all_controllers:
-            if controller.get("type") != controller_type:
+            if controller_type is not None and controller.get("type") != controller_type or\
+               model is not None and model != controller.get("model"):
```

Comment:
please add () to be more clear for condition, also do not use  \\, use () instead.

Reply:
add (), but it seems that \ is still needed, if there is no '\' when switching line, code check won't pass:

(p3-ve) [root@hostA avocado-vt]# inspekt style virttest/libvirt_xml/vm_xml.py
PEP8 disabled: E501,E265,W601,E402
virttest/libvirt_xml/vm_xml.py:578:16: E111 indentation is not a multiple of four
virttest/libvirt_xml/vm_xml.py:578:16: E113 unexpected indentation

(p3-ve) [root@hostA avocado-vt]# inspekt lint virttest/libvirt_xml/vm_xml.py
Pylint disabled: W,R,C,E1002,E1101,E1103,E1120,F0401,I0011
Pylint enabled : W0611
************* Module avocado-vt.virttest.libvirt_xml.vm_xml
E0001:577,0: : invalid syntax (<unknown>, line 577)

Reply:
oh, I know what you mean now

Reply:
updated